tests/migration.go

```diff
@@ -63,6 +61,23 @@ func RunMigration(virtClient kubecli.KubevirtClient, migration *v1.VirtualMachin
 	return migrationCreated
 }
 
+func ConfirmMigrationDataIsStored(virtClient kubecli.KubevirtClient, migration *v1.VirtualMachineInstanceMigration, vmi *v1.VirtualMachineInstance) {
+	By("Retrieving the VMI and the migration object")
+	vmi, err := virtClient.VirtualMachineInstance(vmi.Namespace).Get(vmi.Name, &metav1.GetOptions{})
+	Expect(err).ToNot(HaveOccurred(), "should have been able to retrive the VMI instance")
+	migration, migerr := virtClient.VirtualMachineInstanceMigration(migration.Namespace).Get(migration.Name, &metav1.GetOptions{})
+	Expect(migerr).ToNot(HaveOccurred(), "should have been able to retrive the migration")
+
+	By("Verifying the stored migration state")
+	Expect(migration.Status.MigrationState).ToNot(BeNil(), "should have been able to retrieve the migration `Status::MigrationState`")
+	Expect(vmi.Status.MigrationState.StartTimestamp).To(Equal(migration.Status.MigrationState.StartTimestamp), "the VMI and the migration `Status::MigrationState::StartTimestamp` should be equal")
```

Comment:
Possible nill pointer if `vmi.Status.MigrationState` is nil. I should expect something like
```
Expect(vmi.Status.MigrationState).ToNot(BeNil(), "should have been able to retrieve the vmi `Status::MigrationState`")
```
before check the equality between `migration.Status.MigrationState` and `vmi.Status.MigrationState` values.
Thanks

Reply:
Thanks. And you are right. This is, however, the case already. You can find it in `ConfirmVMIPostMigration`, which is being called right before `ConfirmMigrationDataIsStored`.